### PR TITLE
added helper for resetting default & max leverage value

### PR DESCRIPTION
**Pull request by Blaumaus**
ASANA Ticket: [UI: Feature: AMPF0 max leverage should be x20](https://app.asana.com/0/1125859137800433/1198951310260685/f)

UI Demonstration:
Screenshot from 2021-04-16 14-56-46
Screenshot from 2021-04-16 14-56-56

### CODE REVIEW

File: src/components/OrderForm/OrderForm.helpers.js [hunk 1]
```diff
@@ -130,6 +130,24 @@ const processFieldData = ({ action, layout = {}, fieldData = {} }) => {
...
+  const fields = { ...orderFields }
+  const { BASE, QUOTE } = marketToQuoteBase(market)
+
+  Object.keys(fields).forEach(field => {
+    const { component } = fields[field]
+
+    if (component === 'input.range') {
+      if (BASE === 'AMPF0' && QUOTE === 'USTF0') {
+        fields[field].max = 20
+        fields[field].default = 5
```
Comment (bhoomij) [inline src/components/OrderForm/OrderForm.helpers.js:143]: this should be static and should come from bitfinex api, mostly (https://api.bitfinex.com/v1/symbols_details) can check with ivan also for exact api we can use to retrieve such data

File: src/components/OrderForm/OrderForm.helpers.js [hunk 2]
```diff
@@ -130,6 +130,24 @@ const processFieldData = ({ action, layout = {}, fieldData = {} }) => {
   return data
 }
 
+const fixComponentContext = (orderFields, market) => {
+  const fields = { ...orderFields }
+  const { BASE, QUOTE } = marketToQuoteBase(market)
+
+  Object.keys(fields).forEach(field => {
```
Comment (bhoomij) [inline src/components/OrderForm/OrderForm.helpers.js:137]: can use lodash for loop iteration, _mapkeys or similar

**Review by bhoomij — CHANGES_REQUESTED** [review]

File: src/components/OrderForm/OrderForm.helpers.js (hunk 1 above)
Comment (robertkowalski) [reply src/components/OrderForm/OrderForm.helpers.js:143]: @avsek477 does return it in the available pairs list via websocket

see this PR: https://github.com/bitfinexcom/bfx-hf-server/pull/68

Comment (robertkowalski) [reply src/components/OrderForm/OrderForm.helpers.js:143]: but if possible, using the REST API directly is better, so the Orderform can run without bfx-hf-server :)

**Review by alexstotsky — APPROVED** [review]

**Review by bhoomij — APPROVED** [review]

**Review by bhoomij — APPROVED** [review]
looks good 👍

**Pull request merged**